modules/tracktion_engine/playback/tracktion_DeviceManager.h

```diff
@@ -182,13 +182,16 @@ class DeviceManager     : public juce::ChangeBroadcaster,
 
     Engine& engine;
 
+    int getNumCpuOverloads() { return numCpuOverloads; }
+
 private:
     struct WaveDeviceList;
     struct ContextDeviceClearer;
     bool finishedInitialising = false;
     bool sendMidiTimecode = false;
 
     std::atomic<double> currentCpuUsage { 0 }, streamTime { 0 }, cpuLimitBeforeMuting { 0.98 };
+    std::atomic<int> numCpuOverloads { 0 };
```

Comment:
Is there any time at which this should be reset? In our case, this will continue for all studio sessions within one app session, for example. Maybe we can have a reset API?

Reply:
I believe it's just until the app is closed and everything is closed down. Yeah I think it makes sense to have a reset!